### PR TITLE
Makefile cleanup and image publishing

### PR DESCRIPTION
**Pull request by TerryHowe**
Fixed merge conflicts for Micah's PR

* Added ecr public pre-release model and setup
* Added ecr public cred fetching
* Simplified root Makefile by exporting args

### CODE REVIEW

File: ecr-public/get-credentials.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```
Comment (jaxesn) [inline ecr-public/get-credentials.sh:1]: Want to add the license header to these new scripts?

Comment (TerryHowe) [reply ecr-public/get-credentials.sh:1]: Done

**Review by jaxesn — APPROVED** [review]

**Comment by jaxesn** [comment]
Double check the main-presubmit failure here.

**Comment by TerryHowe** [comment]
It looks like it just timed out getting a go package

**Comment by TerryHowe** [comment]
Merge conflicts fixed again.

**Pull request merged**